### DISCUSSION TITLE
Visual Studio 2022 reports syntax errors in temporary files generated by copilot plugin

### DISCUSSION THREAD

**Discussion by Randname666** [community/community#40593] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Visual Studio reports syntax errors in temporary files generated by copilot plugin which are 8.3 scheme named files under `%LOCALAPPDATA%\temp\github\copilot-vs` which are obviously not the files currently being edited, or files added to solution file or project files. This makes error messages overwhelming and hard to identify actual errors in current file being edited.

Visual Studio considers these files to belong to a project named `CopilotMiscFiles`, despite this project is no where to be found in both currently opened solution files and temporary files folder.
![symptom](https://user-images.githubusercontent.com/38984096/205209847-bb7776da-36c0-4c28-98b3-93ae442b8609.PNG)

**Comment by iamcjwilliams** [comment]
@Randname666 Sorry for the trouble here, Do you have the latest version of the extension? Could you provide a few more details on how this can be replicated?

**Reply by Randname666** [reply]
> @Randname666 Sorry for the trouble here, Do you have the latest version of the extension? Could you provide a few more details on how this can be replicated?

Basically, this happens almost all the time when copilot inline suggestion is used. Most of the time they will dismiss automatically after a few seconds, but not all the time. I haven't found what caused error messages to clog up though.

Also seems those temporary files are never deleted from their temp folder.

**Comment by johnnygerard** [comment]
I have the same issue with version 1.65.7705.

When pressing "Ctrl+Enter" to open GitHub Copilot, I get error messages in the Problems tab.

![image](https://user-images.githubusercontent.com/57420910/211642996-ab2558ef-1cec-44ad-aec4-d8faf5cf7153.png)

**Comment by iamcjwilliams** [comment]
@johnnygerard Have you tried troubleshooting your issue here: https://docs.github.com/en/copilot/troubleshooting-github-copilot?

**Reply by johnnygerard** [reply]
I just checked but the troubleshooting pages only mention logs.
In my case there is nothing logged.

**Reply by iamcjwilliams** [reply]
@johnnygerard Can you provide the steps to replicate the error you're receiving?

**Reply by johnnygerard** [reply]
1. Create an empty directory.
2. Open that folder with Visual Studio Code and click "Yes, I trust the authors" to enable all features.
3. Create a new empty JavaScript file.
4. With GitHub Copilot extension installed (using defaults), type in "function add" (this should be enough to trigger a suggestion).
5. Press "Ctrl+Enter" to open the GitHub Copilot page with detailed suggestions.

Notice how the GitHub Copilot page opens using the current  language mode (in this case JavaScript).

I also tried with TypeScript with similar results but when using Python, there are no syntax errors reported.

**Environment:**
Linux 5.15.0-57-generic
Ubuntu 22.04.1 LTS
Visual Studio Code 1.74.3
GitHub Copilot 1.67.7944

**Reply by iamcjwilliams** [reply]
I have submitted this issue to the copilot team
